client/app/components/form/FormBase.tsx

```diff
@@ -7,6 +7,7 @@ const customFormats = {
   phone: /\(?\d{3}\)?[\s-]?\d{3}[\s-]?\d{4}$/,
   "postal-code":
     /^[ABCEGHJ-NPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][ -]?\d[ABCEGHJ-NPRSTV-Z]\d$/i,
+  bc_corporate_registry_number: "^[A-Za-z]{1,3}\\d{7}$",
```

Comment:
To manually test the ninja schema validation, comment out this line and then try to fill and submit the form with bad data